guides/micronaut-cloud-database-oracle/metadata.json

```diff
@@ -0,0 +1,8 @@
+{
+  "base": "micronaut-cloud-database-base",
+  "title": "Deploy a Micronaut database application to Oracle Cloud",
```

Comment:
Perhaps rename to `"Deploy a Micronaut MySQL database application to Oracle Cloud"` to clarify this is MySQL service